crates/c-api/Cargo.toml

```diff
@@ -18,3 +18,5 @@ doctest = false
 
 [dependencies]
 wasmtime = { path = "../api" }
+wasi-common = { path = "../wasi-common" }
+wasmtime-wasi = { path = "../wasi" }
```

Comment:
Note: built c-api librarary will depend on wasm-common and wasmtime-wasi

Reply:
Was there a concern regarding this comment?

Reply:
nope, just to have it on the record.